firmware/controllers/sensors/map.cpp

```diff
@@ -42,39 +42,43 @@ static float storedInitialBaroPressure = NAN;
  *
  * @returns kPa value
  */
-static FastInterpolation denso183(0, -6.64, 5, 182.78);
-
-/**
- * MAP sensor output voltage of 3.0v = a gauge reading of 0 in. Hg
- * MAP sensor output voltage of 0.5v = a gauge reading of 27 in. Hg
- */
-static FastInterpolation honda3bar(0.5, 91.422, 3.0, 0);
 
-static FastInterpolation subyDenso(0, 0, 5, 200);
+static FastInterpolation mpx4100(0.3, 20, 5, 105);
 
-static FastInterpolation gm3bar(0.631, 40, 4.914, 304);
+static FastInterpolation mpx4115(0, 10, 5, 118);
 
 static FastInterpolation mpx4250(0, 8, 5, 260);
 
 static FastInterpolation mpx4250A(0.25, 20, 4.875, 250);
 
-static FastInterpolation mpx4100(0.3, 20, 4.9, 105);
+static FastInterpolation mpx5700(0, -31, 5, 746);
```

Comment:
can we do something other than this? This is a huge amount of static allocated memory.  Our RAM-poor friends won't be happy :)